src/foam/nanos/bench/BenchmarkResultSystemDAO.js

```diff
@@ -37,6 +37,7 @@ foam.CLASS({
 
       br.setCores(Runtime.getRuntime().availableProcessors());
 
+      Runtime.getRuntime().gc();
```

Comment:
It used to be that you had to do this twice, but I'm not sure that it actually does anything anymore. But better to call twice, just incase.

Reply:
@kgrgreer found this which I think describes the issues. So it's not that two calls are needed but multiple calls to trigger subsequent gc stages.   
"Directly calling System.gc() is always dubious but I can understand why people have seen that multiple calls are required to clean everything up. I don't know if it's still the case but it used to be that objects with finalizers were put in a finalization queue after being garbage collected then the finalizers were actually invoked on the next GC execution. If the objects were still eligible for garbage collection then they were finally deleted after that."

Reply:
Also, when they used a Mark & Sweep GC, you needed two passes, one for the mark and one for the sweep to actually GC things. I'm not sure of the actual reason, but from past experiments, I could see that this was the case, but I'm not sure if this has been the case for a long time.